src/PrestaShopBundle/Twig/LayoutExtension.php

```diff
@@ -85,6 +85,7 @@ public function getGlobals()
 
         try {
             $defaultCurrency = $this->context->getEmployeeCurrency() ?: $this->currencyDataProvider->getDefaultCurrency();
+            $defaultCurrency = $defaultCurrency->getSymbol();
```

Comment:
Are you sure?
Otherwise, the variable name is not the good one :thinking:

Reply:
Fixed the variable naming

Reply:
Add a twig variable default_currency_symbol and return default currency in good variable